package.json

```diff
@@ -14,15 +14,16 @@
   "main": "server/current-context.js",
   "browser": "browser/current-context.js",
   "scripts": {
-    "test": "mocha",
+    "test": "mocha -r cls-hooked",
```

Comment:
It's better to specify options like this in `test/mocha.opts`. That way it still possible to run the tests via `mocha`. I do that frequently, e.g. when I want to run only a subset of tests.